### ISSUE TITLE
List of the 24 building

### ISSUE THREAD

**Issue by agrignard** [CityScope/CS_Cooper-Hewitt#12]
Can we have somewhere on the wiki the list of the 24 buildign with there respective proportion of Residential, office, third places?

Can we have a table with this for instance? 

Building Id | Residential | Office | Third Places
1 | 10% | 20% | 30 %

**Comment by aberke** [comment]
Hello, I have created a spreadsheet to allow the designers and engineers to collaborate on creating the 24 buildings with their proportions of Residential, Office, and Third Places: https://docs.google.com/spreadsheets/d/1aKRPp83EWWdBri6MwjuGcZVSki2xQuxd_OjMAdrrpS4/edit#gid=0
(Please add access for anyone who needs it).

This spreadsheet can be downloaded as a CSV and stored in the repo to replace the current blockData.csv: https://github.com/doorleyr/nyMobility/blob/master/blockData.csv

This block data will be used with the scripts that Ronan developed to generate the simulated population, where that simulated population has their home, work, and third place locations assigned to these blocks.

**Comment by yasushisakai** [comment]
Great, I will let the Design Team know about this.

**Comment by LAAP** [comment]
Hi @aberke, It may be useful to explain what the number on each column means (Number of people, "weight of attraction", etc.), and how it works (higher number = higher production of agents, or similar), So the design team can make decisions based on this simple rules

**Comment by agrignard** [comment]
ok cool let's then update the results of the spreadshett in a wiki in this github

Thanks

**Comment by RELNO** [comment]
@aberke thanks -- please coord this list with Guadalupe  as she work on the 24 types from the tags/modules side.

**Comment by yasushisakai** [comment]
Do we want the same categories as ```blockData.csv```?
we might want a easier name for ```occupationCatN``` fields.

**Comment by agrignard** [comment]
Is this file ready? if yes can someone push it in the github?

**Comment by aberke** [comment]
The original `blockData.csv` file is already accessible and linked to via the submodule `https://github.com/CityScope/CS_activityBased`:
https://github.com/CityScope/CS_activityBased/blob/dd8c378bf05cc7d3fb216c4b0a52e9b2acb615de/blockData.csv

I will change the format a little after speaking with the design team and Yasushi (postponed due to members week activities).

However, this data is not directly needed for the simulation.
This data is an input to the data science analysis that produces the `simPop.csv`.  `simPop.csv` is the data that should be used for the simulation.  It is also already linked to via the submodule (https://github.com/CityScope/CS_activityBased/blob/dd8c378bf05cc7d3fb216c4b0a52e9b2acb615de/results/simPop.csv)

I think you are already able to start working with the `simPop.csv`, no?

Once the `blockData.csv` is updated, the `simPop.csv` data will change as well, but not the format.
If the format of `simPop.csv` needs to change, do you mind starting another issue?

**Comment by aberke** [comment]
The names of the occupational categories is documented in the README for the https://github.com/CityScope/CS_activityBased repo.

@yasushisakai The occupational categories are relevant to the work places assigned to the produced simPop population.  I recommend that we keep these categories, but I will update the spreadsheet to use these english names (vs `occupationCat_N`) and update the code accordingly.

This will allow the design team to fill in the block data based on the work places/amenities they designed into the blocks.

**Comment by aberke** [comment]
@yasushisakai (and others)
I have created a tab in this spreadsheet that I would like to propose for collaborating with the design team to create the blocks:
https://docs.google.com/spreadsheets/d/1aKRPp83EWWdBri6MwjuGcZVSki2xQuxd_OjMAdrrpS4/edit#gid=919292076

For each block (one block per row in spreadsheet) the design team can say determine the number for

- Residential: The number of people who live in this block
- Workplace types: For each of the 5 categories of work, how many people do work of that type in this block.  (The workplace categories were determined by the NHTS data)
  - Sales or service
  - Clerical or administrative
  - Manufacturing, construction, maintenance, or farming
  - Professional, managerial, or technical
  - Student
- Other places: The number of people who might go here as "other places" such as a place to get a coffee, go for a stroll, or do any "other" activities during the day that are not going home or or going to work

This spreadsheet can then be downloaded as a CSV to produce the simulated population data, where the agents are assigned to blocks for work and home locations.

If you think this is a good format and process, I can update the code to ingest this new data and document it.  Please comment on how you would like to see this tweaked beforehand.

**Comment by yasushisakai** [comment]
@aberke, thanks! 
Just a heads up to people outside the design team, 
the sim team has synced on this with the design team.

**Comment by agrignard** [comment]
Could be nice to have a computational version of the spreadsheet. I guess that the google doc is usefull to put some information collaboratively but having a simple .csv in the github will be needed at some point for the simulation side

I will produce one soon, and then someone need to make the link between the google doc and the "real" file

**Comment by aberke** [comment]
Yes, the plan is to download it as the `blocks.csv` data to store/commit in github once it is ready by the design team.

**Comment by agrignard** [comment]
@yasushisakai and @aberke if you are interested to follow the progression of the code here is the commit that integrate those value in the building

https://github.com/CityScope/CS_Cooper-Hewitt/commit/39f7e4b7d902f91c306706df63b51106d94b3f4f

I have slightly modify the google spreadsheet header so that ones only has to export it as a csv and replace it here if there is any change
https://github.com/CityScope/CS_Cooper-Hewitt/tree/master/ABMobility/data/block

I close this issue as the file is now integrated in the simualtion, however it might need some modification if it's the case feel free to reopen or recreate another issue. 

As some point it will be also important to start to populate the wiki instead of using the issue as a documentation place holder

**Comment by RELNO** [comment]
@agrignard just making sure we're on the same page: 
https://github.com/CityScope/CS_Cooper-Hewitt/commit/39f7e4b7d902f91c306706df63b51106d94b3f4f#diff-0ce9d977d51aaa959355969ce56cb2b2
has buildings starting with `1` to `24`, where the scanner has `0-23`

**Comment by agrignard** [comment]
yes the design team are more in the design style using 1 for a first object. I took care of the bridge. The scanning is sending 0 to 23 but we display it on the table as 1 to 24 so it matches with the sticker and the design team. Welcome to the CS vs Design

**Comment by RELNO** [comment]
![image](https://user-images.githubusercontent.com/14296774/47623461-4930c180-dae8-11e8-8fe5-fc97081a432d.png)

**Comment by aberke** [comment]
Thank you for sharing the change @agrignard !
